source/_docs/wordpress-s3.md

```diff
@@ -0,0 +1,113 @@
+
+## Configure S3 within the AWS Console
+Before integrating S3 with your site, you'll need to configure the service within your [AWS Management Console](https://console.aws.amazon.com){.external}.
+
+### Create a New AWS S3 Bucket
+If you do not have an existing bucket for your site, create one:
+
+1. From your [AWS Console](https://console.aws.amazon.com){.external}, click **S3**.
+2. Click **Create Bucket**.
+3. Enter a bucket name. The bucket name you choose must be unique across all existing bucket names in Amazon S3.
+
+    <div class="alert alert-info" role="alert">
+    <h4 class="info">Note</h4>
+    <p>After you create a bucket, you cannot change its name. The bucket name is visible in the URL that points to the objects stored in the bucket. Ensure that the bucket name you choose is appropriate.</p>
+    </div>
+
+4. Select a region and click **Create**.
+5. The **Set properties** section has additional configuration options you can configure now, or wait and configure later. When complete, click **Next**.
+6. In the **Permissions** tab tick the boxes for **Read** and **Write** access for both **Objects** and **Permissions**, then click **Save**.
+7. Review your settings, then click **Create bucket**.
+
+## Integrate S3 with WordPress
+You will need to install a plugin such as [S3 Uploads](https://github.com/humanmade/S3-Uploads){.external} or [WP Offload S3](https://deliciousbrains.com/wp-offload-s3/){.external}.
+
+WP Offload S3 requires a paid license but is configurable in the WordPress admin UI and offers a number of options and features. S3 Uploads is open-source but does not include an admin UI and requires [Terminus](/docs/terminus) and [WP-CLI](/docs/wp-cli) for setup and migration.
+
+### Install and Deploy S3 Uploads
+
+<div class="alert alert-info" role="alert">
+  <h4 class="info">Note</h4>
+  <p markdown="1">This plugin currently conflicts with [Solr Power](https://wordpress.org/plugins/solr-power/){.external}, our recommended plugin for Solr integration. [More info](https://github.com/humanmade/S3-Uploads/issues/80){.external}.</p>
+</div>
+
+1. Download the latest plugin release from [Github](https://github.com/humanmade/S3-Uploads/releases){.external} and extract it to `wp-content/plugins/`. Note that our documentation has been tested for version 2.0.0.
+
+    <div class="alert alert-danger" role="alert">
+    <h4 class="info">Warning</h4>
+    <p markdown="1">**Do not** add the plugin as a Git submodule. Git submodules are not supported on the platform ([more info](https://pantheon.io/docs/git-faq/#does-pantheon-support-git-submodules)).</p>
+    </div>
+
+2. Rename the extracted folder to remove the version number. For example:
+
+    ```bash
+    mv S3-Uploads-2.0.0/ S3-Uploads
+    ```
+
+3. Create and / or copy your S3 upload key and secret from the "My security credentials" section of your AWS account to a text editor on your local computer.
```

Comment:
@sarahg I think this is where I'm getting stuck. I cannot find any "Upload Key" information on AWS. 

While on the subject, it seems like AWS is strongly pushing me to have an "IAM" user to associate with this setup. Thoughts?

Reply:
@alexfornuto Good catch! The plugin refers to them a bit differently than the AWS UI does. Adjusted the label in 4acec05, can you see if that makes it clearer?

As for the IAM user — that gets created via a WP-CLI command a little further down the line (`terminus wp <site>.<env> -- s3-uploads create-iam-user`). Might be worth noting that at this point, too, though?